Recrutify/Recrutify.Services/Validators/UpdateProjectValidator.cs

```diff
@@ -31,21 +31,28 @@ private void ConfigureRules()
         private async Task UmmutableFieldsAreKeepingUnchanged(UpdateProjectDTO dto, ValidationContext<UpdateProjectDTO> context, CancellationToken cancellationToken)
         {
             var project = await _projectRepository.GetAsync(dto.Id);
-            var currentPrimarySkillsIds = project.PrimarySkills.Select(x => x.Id).ToList();
-            var primarySkillIds = dto.PrimarySkills.Select(x => x.Id).ToList();
-            if (dto.Name != project.Name)
+            if (project == null)
             {
-                context.AddFailure("Name cannot be changed!");
+                context.AddFailure("Project doesn't exist");
```

Comment:
you can just write **return;**
and remove if/else

Reply:
done